src/earn/EarnEnterAmount.tsx

```diff
@@ -579,15 +582,40 @@ function FeeDetailsBottomSheet({
   feeCurrency,
   estimatedFeeAmount,
   maxFeeAmount,
+  swapTransaction,
+  pool,
+  token,
 }: {
   forwardedRef: React.RefObject<BottomSheetModalRefType>
   title: string
   testID: string
   feeCurrency?: TokenBalance
   estimatedFeeAmount?: BigNumber
   maxFeeAmount?: BigNumber
+  swapTransaction?: SwapTransaction | undefined
+  pool: EarnPosition
+  token: TokenBalance
 }) {
   const { t } = useTranslation()
+  const depositToken = useTokenInfo(pool.dataProps.depositTokenId)
+
+  if (!depositToken) {
+    // should never happen
+    throw new Error(`Token info not found for token ID ${pool.dataProps.depositTokenId}`)
+  }
+
+  const swapFeeAmount = useMemo(() => {
+    if (swapTransaction && swapTransaction.appFeePercentageIncludedInPrice) {
+      return new BigNumber(swapTransaction.buyAmount)
```

Comment:
`buyAmount` is the swap to amount. Can we use the user inputted amount (tokenAmount from the parent component)?

Reply:
This is done in #6026, but I'll add those changes here.

Reply:
Fixed in 4431acd89efe946232c434c2c51bb5d0482fe57a!